h/static/scripts/directive/annotation.coffee

```diff
@@ -202,6 +208,10 @@ AnnotationController = [
       switch @action
         when 'create'
           updateDomainModel(model, @annotation)
+
+          if group.focusedGroup().hashid != '__public__'
```

Comment:
This is confusing. How could the hashid be a readable string like `__public__`. Should the client even know these are hashids? Shouldn't they just be opaque identifiers here, and the `hash` bit is an implementation detail hidden within the server?